test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -14,14 +15,11 @@
 // Disable magic number check to make defaults readable
 @SuppressWarnings("checkstyle:magicnumber")
 public class ServerContextBuilder {
-
-    long initialToken = 0L; // for testing, we want to reset the sequencer on each test
-
     boolean single = true;
     boolean memory = true;
     String logPath = null;
-    boolean noVerify = false;
-    boolean noSync = false;
+    boolean verifyChecksum = true;
+    boolean syncData = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651442)